fix(Logs in context): Clarified that it's the agent configuration that needs to be updated (#16196)
newrelic/docs-website: src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx

 
  Using this option, your logs include `span.id`, `trace.id`, `hostname`, `entity.guid`, and `entity.name`. [Learn about log forwarding limitations.](/docs/logs/logs-context/get-started-logs-context#forwarding-details)

>   All you need to do is install an agent version with log forwarding capabilities ([.NET agent 9.7.0 or higher](/docs/release-notes/agent-release-notes/net-release-notes)). If forwarding is disabled, you can use this to update your agent configuration to enable it:

Reviewer: I'm not sure if I'm missing something so I'm not just making this change manually, but should this be "to update your agent configuration and enable it:"? 

"to update... to enable" reads a little awkwardly in sequence

Author: I'm trying to clarify that it's the agent configuration that people need to update. (We've gotten more than one piece of customer feedback that the old wording is confusing on this point.) If you have a better alternative, feel free to update it.

Author: @jeff-colucci what do you think?

Reviewer: Sorry for some reason your initial comment didn't get put in my Slack. Bad Github bot!

What about this: "If forwarding is disabled, you can use this to enable forwarding and update your agent configuration"?

I guess the confusion is on what the "this" in "you can use this" is in either sentence. What do you think @paperclypse

Author: I worry that that confuses things, because it's updating the agent configuration that enables the forwarding. The "this" is the configuration snippet. 

Maybe this would be better? 

"If forwarding is disabled, you can enable it by adding the following to your agent configuration file:"

Author: (I'm still not clear on if you have to ping someone directly or not to be notified in GitHub @jeff-colucci :) )

Reviewer: I like that suggestion a lot. Clear and concise and does what I want it to as a reader! @paperclypse

Author: Done! I'll do a squash merge once you approve it :) (I'm deliberately not pinging you here, because I want to see if it will notify you)

>   All you need to do is install an agent version with log forwarding capabilities ([.NET agent 9.7.0 or higher](/docs/release-notes/agent-release-notes/net-release-notes)). If forwarding is disabled, you can use this to update your agent configuration to enable it:

Author: ```suggestion
  All you need to do is install an agent version with log forwarding capabilities ([.NET agent 9.7.0 or higher](/docs/release-notes/agent-release-notes/net-release-notes)). If forwarding is disabled, you can enable it by adding the following to your agent configuration file::
```
